functions/to.fish

```diff
@@ -68,6 +69,22 @@ function __to_ls
   end
 end
 
+function __to_here
+    set -x currentdir (pwd)
+    set -x wholepath (ls -l ~/.tofish | grep $currentdir | cut -d':' -f 2 | cut -d' ' -f 2,4 )
+    for line in $wholepath
+        set -x inner (echo $line | cut -d' ' -f 2)
```

Comment:
Use fish's native [string function](https://fishshell.com/docs/current/cmds/string.html) instead of `cut`

Reply:
I haven't found a way to keep two fields together, like in line 74 the second 'cut' keeps fields 2 and 4 to get the simlink and the subdir. When the inner loop finds the dir(s) from field 4 of that cut, it just uses field 2 as the answer.